public/index.html

```diff
@@ -50,9 +49,6 @@
     <meta property="og:image:height" content="669" />
     <meta property="og:image:alt" content="Excalidraw logo with byline." />
 
-    <!-- Safari -->
-    <meta name="apple-mobile-web-app-status-bar-style" content="default" />
```

Comment:
Great find, @j-f1, it’s right there in the docs even: 

> “If set to black-translucent, the web content is displayed on the entire screen, partially obscured by the status bar. The default value is default.”— https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html

I’ll file an Issue to get this fixed upstream in the library.

Reply:
Actually, https://github.com/GoogleChromeLabs/pwacompat/issues/26 suggests my initial [suspicion](https://github.com/excalidraw/excalidraw/pull/1433#issuecomment-613987183) that we’re holding the notch CSS wrong maybe still holds. I’ll trial and error some more tomorrow. It would look gorgeous if it flowed around the notch elegantly.